tools/wptserve/wptserve/server.py

```diff
@@ -511,6 +634,12 @@ def __init__(self, headers):
             self.raw_headers[key] = val
             dict.__setitem__(self, self._convert_h2_header_to_h1(key), val)
 
+    def get(self, key, def_value=None):
+        # pywebsocket will read the upper case value of those headers
```

Comment:
This seems a bit brittle. Are headers in HTTP/2 also case-insensitive? If so, can we do case-insensitive lookup for all headers similar to what you can find in `request.py`? If not, this seems like a bug elsewhere that needs to be fixed.

Reply:
HTTP/2 headers are case sensitive. I will modify the handshake code to read the correct (lower case) headers instead.